hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestDeletedBlockRetryCountRenewer.java

```diff
@@ -0,0 +1,166 @@
+import java.util.stream.Collectors;
+
+import static org.apache.hadoop.hdds.scm.ScmConfigKeys.OZONE_SCM_BLOCK_DELETION_MAX_RETRY;
+
+/**
+ * Test for DeletedBlockRetryCountRenewer Cli.
+ */
+public class TestDeletedBlockRetryCountRenewer {
+
+  private static final Logger LOG = LoggerFactory
+      .getLogger(TestDeletedBlockRetryCountRenewer.class);
+  private MiniOzoneHAClusterImpl cluster = null;
+  private OzoneConfiguration conf;
+  private String clusterId;
+  private String scmId;
+  private String scmServiceId;
+  private int numOfSCMs = 3;
+
+  /**
+   * Create a MiniOzoneHACluster for testing.
+   *
+   * @throws IOException
+   */
+  @BeforeEach
+  public void init() throws Exception {
+    conf = new OzoneConfiguration();
+    clusterId = UUID.randomUUID().toString();
+    scmId = UUID.randomUUID().toString();
+    scmServiceId = "scm-service-test1";
+
+    conf.setBoolean(ScmConfigKeys.OZONE_SCM_HA_ENABLE_KEY, true);
+    conf.setInt(OZONE_SCM_BLOCK_DELETION_MAX_RETRY, 20);
+
+    cluster = (MiniOzoneHAClusterImpl) MiniOzoneCluster.newOMHABuilder(conf)
+        .setClusterId(clusterId)
+        .setScmId(scmId)
+        .setSCMServiceId(scmServiceId)
+        .setNumOfStorageContainerManagers(numOfSCMs)
+        .setNumOfActiveSCMs(numOfSCMs)
+        .setNumOfOzoneManagers(1)
+        .build();
+    cluster.waitForClusterToBeReady();
+  }
+
+  /**
+   * Shutdown MiniDFSCluster.
+   */
+  @AfterEach
+  public void shutdown() {
+    if (cluster != null) {
+      cluster.shutdown();
+    }
+  }
+
+  //<containerID,  List<blockID>>
+  private Map<Long, List<Long>> generateData(int dataSize) {
+    Map<Long, List<Long>> blockMap = new HashMap<>();
+    Random random = new Random(1);
+    int continerIDBase = random.nextInt(100);
+    int localIDBase = random.nextInt(1000);
+    for (int i = 0; i < dataSize; i++) {
+      long containerID = continerIDBase + i;
+      List<Long> blocks = new ArrayList<>();
+      for (int j = 0; j < 5; j++)  {
+        long localID = localIDBase + j;
+        blocks.add(localID);
+      }
+      blockMap.put(containerID, blocks);
+    }
+    return blockMap;
+  }
+
+  private StorageContainerManager getSCMLeader() {
+    return cluster.getStorageContainerManagersList()
+        .stream().filter(a -> a.getScmContext().isLeaderReady())
+        .collect(Collectors.toList()).get(0);
+  }
+
+  @Test
+  public void testRenewCmd() throws IOException, TimeoutException {
+    int maxRetry = conf.getInt(OZONE_SCM_BLOCK_DELETION_MAX_RETRY, 20);
+    // add some block deletion transactions
+    DeletedBlockLog deletedBlockLog = getSCMLeader().
+        getScmBlockManager().getDeletedBlockLog();
+    deletedBlockLog.addTransactions(generateData(30));
+    getSCMLeader().getScmHAManager().asSCMHADBTransactionBuffer().flush();
+    LOG.info("Valid num of txns: {}", deletedBlockLog.
+        getNumOfValidTransactions());
+    Assertions.assertEquals(deletedBlockLog.getNumOfValidTransactions(), 30);
```

Comment:
Nit: `assertEquals` arguments should be swapped (`expected`, `actual`) in all assertions.

https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertEquals(int,int)

Reply:
Got it!